smartsim/_core/_install/configs/mlpackages/LinuxX64CUDA125.json

```diff
@@ -0,0 +1,64 @@
+{
+    "platform": {
+        "operating_system":"linux",
+        "architecture":"x86_64",
+        "device":"cuda-12.5"
+    },
+    "ml_packages": [
+        {
+            "name": "dlpack",
+            "version": "v0.5_RAI",
+            "pip_index": "",
+            "python_packages": [],
+            "lib_source": "https://github.com/RedisAI/dlpack.git"
+        },
+        {
+            "name": "libtorch",
+            "version": "2.4.0",
+            "pip_index": "https://download.pytorch.org/whl/cu124",
+            "python_packages": [
+                "torch<=2.4.0",
+                "torchvision<=0.19.0",
+                "torchaudio<=2.4.0"
```

Comment:
At time of writing, from a clean build, these version numbers will currently throw a warning message during the install process despite the fact that the expected version of the package was installed. This is because the [check to make sure the appropriate python packages is installed looks for a strict match of any version suffix](https://github.com/CrayLabs/SmartSim/pull/669/commits/9cefd75c77fe417d0e3d8b957916642873fee805#diff-5b4ed2e1cb9524b32297844cf6e58d61fac9293474767fcb1c701e66e574e53fR162-R167) between the requested and found versions.

We should be able to fix this by appending `+cuda` to the version numbers here, but I wasn't sure if that this is the intended behavior. I just wanted to go ahead and call out this issue while I'm here and we can discuss how we want to resolve the issue.

A similar problem exists for the other gpu and linux+cpu builds of `torch`.

Reply:
Yes, I think I did this for the 11.8. We should just update the JSON files to be clear about the variant.